### PR TITLE
Luke/use glamor in button

### PR DESCRIPTION
**Pull request by lukeschunk**
This PR 
* Adds glamor as a dependency 
* Uses Glamor in the Button component ( I think this will fix a bug internally )

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -66,9 +67,9 @@ class Button extends React.Component {
 
     return (
       <button
+        className={css(Object.assign({}, styles.component, styles[this.props.type], style))}
```
Comment (cerinman) [inline src/components/Button.js]: I know the object assign was already there but could you switch it object spread while you're here?

File: src/components/Button.js [hunk 2]
```diff
@@ -248,4 +249,4 @@ class Button extends React.Component {
   };
 }
 
-module.exports = Radium(Button);
+module.exports = Button;
```
Comment (cerinman) [inline src/components/Button.js]: Do you think that removing `Radium` will fix the bug you are experiencing?

Comment (lukeschunk) [reply src/components/Button.js]: i do

File: src/components/Button.js (hunk 1 above)
Comment (lukeschunk) [reply src/components/Button.js]: 👍

**Comment by cerinman** [comment]
:shipit:

**Pull request merged**